src/main/java/com/google/sps/servlets/TasksServlet.java

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.services.tasks.Tasks;
+import com.google.api.services.tasks.model.Task;
+import com.google.api.services.tasks.model.TaskList;
+import com.google.appengine.repackaged.com.google.gson.Gson;
+import com.google.sps.utility.AuthenticationUtility;
+import com.google.sps.utility.TasksUtility;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Serves selected information from the User's Tasks Account. TODO: Create Servlet Utility to handle
+ * common functions (Issue #26)
+ */
+@WebServlet("/tasks")
+public class TasksServlet extends HttpServlet {
```

Comment:
Are you going to add tests for this class?

Reply:
This will be changed in another PR #39

Reply:
This will be handled in the next PR #39